frigate/output.py

```diff
@@ -310,11 +315,14 @@ def update_frame(self):
 
         return True
 
-    def update(self, camera, object_count, motion_count, frame_time, frame) -> bool:
+    def update(self, camera, camera_config, object_count, motion_count, frame_time, frame) -> bool:
+        # don't process if birdseye is disabled for this camera
+        if camera_config and not camera_config.enabled:
+            return False
 
         # update the last active frame for the camera
         self.cameras[camera]["current_frame"] = frame_time
-        if self.camera_active(object_count, motion_count):
+        if self.camera_active(camera_config, object_count, motion_count):
```

Comment:
Just pass `camera_config.mode`. The config parser should ensure that that value is always set.